src/main/java/org/libsdl/app/SDLActivity.kt

```diff
@@ -177,7 +181,7 @@ open class SDLActivity internal constructor (context: Context?) : RelativeLayout
 
     override fun doFrame(frameTimeNanos: Long) {
         if (isRunning && mIsSurfaceReady) {
-            this.nativeRender()
+            this.nativeRenderAndProcessEvents()
```

Comment:
this should probably be called nativeProcessEventsAndRender (because it happens in that order)